teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -680,6 +682,12 @@ interface FinishBuildTrigger {
     val excludedBranchPatterns: Set<String>
 }
 
+interface SnapshotDependency {
+    val id: SnapshotDependencyId
```

Comment:
Id of a snapshot dependency is always the same as configuration id. Maybe it can be removed?